apps/base-docs/tutorials/docs/0_gassless-transactions-with-paymaster.md

```diff
@@ -0,0 +1,616 @@
+
+## Prerequisites
+
+This tutorial assumes you have a Coinbase Cloud Developer Platform account. If not, sign up [here](https://portal.cdp.coinbase.com/).
+
+- **Coinbase CDP account:** This is your access point to the Coinbase Cloud Developer Platform, where you can manage projects and utilize tools like the Paymaster.
+- **Familiarity with Smart Accounts and ERC 4337:** Understanding Smart Accounts and the ERC 4337 standard is crucial as they are the backbone of executing advanced transaction patterns and account abstractions on the Ethereum network.
+- **Rust:** Rust is a systems programming language focused on safety and performance. You will need basic knowledge of Rust for some coding aspects of this tutorial.
+- **Foundry:** Foundry is a development environment, testing framework, and smart contract toolkit for Ethereum. It's essential for deploying and testing smart contracts.
+
+### Setup a Coinbase Paymaster & Bundler
+
+In this section, you will configure a Paymaster to sponsor payments on behalf of a specific smart contract for a specified amount. First, navigate to the Coinbase Developer Platform, create or select your project, and click on the Paymaster tool from the left navigation. Then, go to the Configuration tab and save the RPC URL to your clipboard, which will be needed for later steps in your index.js file.
+
+**Navigate to the** **[Coinbase Developer Platform](https://portal.cdp.coinbase.com/)**:
+
+Create or select your project of choice from the upper left corner of your screen.
+
+![cdp-home.png](../../assets/images/gasless-transaction-on-base/cdp-select-project.png)
+
+**Click on the `Paymaster` tool on the left navigation:** [Paymaster Tool](https://portal.cdp.coinbase.com/products/bundler-and-paymaster)
+
+![cdp-paymaster-tool.png](../../assets/images/gasless-transaction-on-base/cdp-paymaster.png)
+
+**Click on `Configuration` at the top of the screen**
+
+![cdp-paymaster-tool.png](../../assets/images/gasless-transaction-on-base/cdp-config.png)
+
+**Save the RPC URL to your paymaster to your clipboard:** You will need it in your `index.js` file in a later step.
+
+### Allowlist a Sponsorable Contract
+
+Sponsoring transactions are beneficial to easing the onboarding and UX of a decentralized application. As a developer, you want to ensure this is done in the most secure and cost-effective manner. Start by allowlisting a contract of your choice.
+
+**Select Base Mainnet**
+
+From the configuration page, select `Base Mainnet` from the dropdown menu. Then, enable your paymaster by clicking on the toggle button to the right of the screen.
+
+![cdp-project-selection.png](../../assets/images/gasless-transaction-on-base/cdp-select-project.png)
+
+**Allowlist the NFT contract and the mintTo functions:**
+
+Click the `Add` button to add a contract.
+
+Add the following contract: `0x83bd615eb93eE1336acA53e185b03B54fF4A17e8`
+
+The function to enable is `mintTo`
```

Comment:
It looks like you need to click `Save` on the main page after you do this?  I would bold that one, it's easy to miss!